docs/types/txikijs.d.ts

```diff
@@ -944,17 +929,17 @@ declare global {
         const ppid: number;
         
         interface UserInfo {
-            username: string;
+            userName: string;
             uid: number;
```

Comment:
For consistency maybe having `userId` and `groupId` in place of those shorts acronyms would be better

Reply:
👍